test/e2e/storage/testsuites/util.go

```diff
@@ -0,0 +1,40 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package testsuites
+
+import (
+	v1 "k8s.io/api/core/v1"
+	"k8s.io/kubernetes/test/e2e/framework"
+	e2epv "k8s.io/kubernetes/test/e2e/framework/pv"
+	imageutils "k8s.io/kubernetes/test/utils/image"
+)
+
+// TODO: after issue https://github.com/kubernetes/kubernetes/issues/81245 is resolved
+// this utility can be moved to e2epod
+func getTestImage() int {
```

Comment:
Sorry why is this related to https://github.com/kubernetes/kubernetes/issues/81245?
I think we can use utils from e2e/framework not the other way around?